apps/vr-tests-react-components/.storybook/preview.js

```diff
@@ -1,72 +1,4 @@
 // @ts-check
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31939&commitId=dc71f6c1d48aa0ac1c859d476ea4834c95057a24&env=prod&clientType=FLUENTUI)


  <details><summary><b>Accordion Converged</b> 25 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Accordion Converged.disabled - High Contrast.normal.chromium.png|1848| Changed|
|Accordion Converged.disabled.normal.chromium.png|9720| Changed|
|Accordion Converged.expandIcon   Icon-   - High Contrast.normal.chromium.png|2386| Changed|
|Accordion Converged.expandIcon   Icon-  .normal.chromium.png|11060| Changed|
|Accordion Converged.expandIconPosition  end .normal.chromium.png|9518| Changed|
|Accordion Converged.disabled - Dark Mode.normal.chromium.png|1979| Changed|
|Accordion Converged.expandIconPosition  end  - RTL.normal.chromium.png|9519| Changed|
|Accordion Converged.expandIcon   Icon-   - Dark Mode.normal.chromium.png|3441| Changed|
|Accordion Converged.expandIcon   Icon-   - RTL.normal.chromium.png|11063| Changed|
|Accordion Converged.visibility+focus - RTL.normal.chromium.png|9565| Changed|
|Accordion Converged.visibility+focus - High Contrast.focus closed.chromium.png|2468| Changed|
|Accordion Converged.visibility+focus - Dark Mode.normal.chromium.png|2380| Changed|
|Accordion Converged.visibility+focus - High Contrast.focus opened.chromium.png|1208| Changed|
|Accordion Converged.icon   Icon-   - RTL.normal.chromium.png|10330| Changed|
|Accordion Converged.visibility+focus.focus closed.chromium.png|9986| Changed|
|Accordion Converged.visibility+focus.normal.chromium.png|9566| Changed|
|Accordion Converged.icon   Icon-  .normal.chromium.png|10330| Changed|
|Accordion Converged.visibility+focus - High Contrast.normal.chromium.png|1208| Changed|
|Accordion Converged.visibility+focus - RTL.focus closed.chromium.png|9985| Changed|
|Accordion Converged.visibility+focus.focus opened.chromium.png|9566| Changed|
|Accordion Converged.visibility+focus - RTL.focus opened.chromium.png|9565| Changed|
|Accordion Converged.visibility+focus - Dark Mode.focus opened.chromium.png|2380| Changed|
|Accordion Converged.visibility+focus - Dark Mode.focus closed.chromium.png|3946| Changed|
|Accordion Converged.size.normal.chromium.png|14198| Changed|
|Accordion Converged.size - RTL.normal.chromium.png|14199| Changed|

</details>


  <details><summary><b>Avatar Converged</b> 21 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.basic - Dark Mode.normal.chromium.png|3108| Changed|
|Avatar Converged.basic.normal.chromium.png|14676| Changed|
|Avatar Converged.image-bad-url+icon.normal.chromium.png|1726| Changed|
|Avatar Converged.basic - High Contrast.normal.chromium.png|3047| Changed|
|Avatar Converged.image-bad-url.normal.chromium.png|1763| Changed|
|Avatar Converged.basic - RTL.normal.chromium.png|14678| Changed|
|Avatar Converged.color - Dark Mode.normal.chromium.png|11835| Changed|
|Avatar Converged.customSize+name+badge.normal.chromium.png|19454| Changed|
|Avatar Converged.customSize+icon+active.normal.chromium.png|23037| Changed|
|Avatar Converged.color - High Contrast.normal.chromium.png|17385| Changed|
|Avatar Converged.customSize+image.normal.chromium.png|25499| Changed|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|44850| Changed|
|Avatar Converged.badgeMask.normal.chromium.png|44831| Changed|
|Avatar Converged.size+inactive+badge.normal.chromium.png|32738| Changed|
|Avatar Converged.color+active - High Contrast.normal.chromium.png|31546| Changed|
|Avatar Converged.color+active.normal.chromium.png|46122| Changed|
|Avatar Converged.color.normal.chromium.png|36832| Changed|
|Avatar Converged.size+image+badge.normal.chromium.png|39612| Changed|
|Avatar Converged.size+name.normal.chromium.png|24644| Changed|
|Avatar Converged.color+active - Dark Mode.normal.chromium.png|20463| Changed|
|Avatar Converged.size+icon+badge+square.normal.chromium.png|26318| Changed|

</details>


  <details><summary><b>Card Converged</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged.card templates.normal.chromium.png|138991| Changed|
|Card Converged.CardHeader.normal.chromium.png|124796| Changed|
|Card Converged.size.normal.chromium.png|216065| Changed|
|Card Converged.appearance - High Contrast.normal.chromium.png|62942| Changed|
|Card Converged.appearance.normal.chromium.png|355846| Changed|
|Card Converged.appearance - RTL.normal.chromium.png|326123| Changed|
|Card Converged.appearance - Dark Mode.normal.chromium.png|83465| Changed|
|Card Converged.orientation.chromium.png|16475| Changed|

</details>


  <details><summary><b>Card Converged - Interactive</b> 36 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.normal.chromium.png|1519598| Changed|
|Card Converged - Interactive.appearance interactive - Filled - Dark Mode.hover.chromium.png|1576538| Changed|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.hover.chromium.png|1699027| Changed|
|Card Converged - Interactive.appearance interactive - Filled - Dark Mode.click.chromium.png|1545324| Changed|
|Card Converged - Interactive.appearance interactive - Filled - Dark Mode.normal.chromium.png|1556868| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - Dark Mode.hover.chromium.png|1565969| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - High Contrast.hover.chromium.png|1699027| Changed|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.click.chromium.png|1699027| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - Dark Mode.normal.chromium.png|1546549| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - High Contrast.normal.chromium.png|1519598| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - High Contrast.click.chromium.png|1699027| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - Dark Mode.click.chromium.png|1534229| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative.hover.chromium.png|1768148| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative.click.chromium.png|1745322| Changed|
|Card Converged - Interactive.appearance interactive - Filled.click.chromium.png|1750755| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative.normal.chromium.png|1779447| Changed|
|Card Converged - Interactive.appearance interactive - Filled.hover.chromium.png|1773241| Changed|
|Card Converged - Interactive.appearance interactive - Filled.normal.chromium.png|1785160| Changed|
|Card Converged - Interactive.appearance interactive - Outline - Dark Mode.normal.chromium.png|1559235| Changed|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.click.chromium.png|1700657| Changed|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.hover.chromium.png|1700657| Changed|
|Card Converged - Interactive.appearance interactive - Outline.hover.chromium.png|1785363| Changed|
|Card Converged - Interactive.appearance interactive - Outline.click.chromium.png|1785220| Changed|
|Card Converged - Interactive.appearance interactive - Outline - Dark Mode.click.chromium.png|1559302| Changed|
|Card Converged - Interactive.appearance interactive - Outline - Dark Mode.hover.chromium.png|1559435| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - Dark Mode.hover.chromium.png|1574122| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - Dark Mode.normal.chromium.png|1558450| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - High Contrast.click.chromium.png|1701682| Changed|
|Card Converged - Interactive.appearance interactive - Outline.normal.chromium.png|1785436| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - Dark Mode.click.chromium.png|1563614| Changed|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.normal.chromium.png|1521097| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - High Contrast.hover.chromium.png|1701682| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - High Contrast.normal.chromium.png|1521097| Changed|
|Card Converged - Interactive.appearance interactive - Subtle.hover.chromium.png|1774400| Changed|
|Card Converged - Interactive.appearance interactive - Subtle.normal.chromium.png|1785771| Changed|
|Card Converged - Interactive.appearance interactive - Subtle.click.chromium.png|1752077| Changed|

</details>


  <details><summary><b>Card Converged - Selectable</b> 68 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Selectable.appearance focusable + selectable.click.chromium.png|16921| Changed|
|Card Converged - Selectable.appearance focusable + selectable.hover.chromium.png|17294| Changed|
|Card Converged - Selectable.appearance focusable + selectable.normal.chromium.png|17967| Changed|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.click.chromium.png|8012| Changed|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.hover.chromium.png|8146| Changed|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.selected.chromium.png|8528| Changed|
|Card Converged - Selectable.appearance focusable + selectable.selected.chromium.png|17380| Changed|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.normal.chromium.png|7901| Changed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.selected.chromium.png|12032| Changed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.normal.chromium.png|11812| Changed|
|Card Converged - Selectable.appearance selectable - Filled - RTL.click.chromium.png|16928| Changed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.click.chromium.png|12668| Changed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.hover.chromium.png|12668| Changed|
|Card Converged - Selectable.appearance selectable - Filled - RTL.normal.chromium.png|17975| Changed|
|Card Converged - Selectable.appearance selectable - Filled - RTL.hover.chromium.png|17301| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - Dark Mode.click.chromium.png|8727| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - Dark Mode.normal.chromium.png|8055| Changed|
|Card Converged - Selectable.appearance selectable - Filled - RTL.selected.chromium.png|17387| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - Dark Mode.selected.chromium.png|8338| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - Dark Mode.hover.chromium.png|7873| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - High Contrast.hover.chromium.png|12668| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - High Contrast.click.chromium.png|12668| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.normal.chromium.png|17684| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - High Contrast.selected.chromium.png|12032| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.hover.chromium.png|17343| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.hover.chromium.png|17336| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.click.chromium.png|16790| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - High Contrast.normal.chromium.png|11812| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.click.chromium.png|16796| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.normal.chromium.png|17676| Changed|
|Card Converged - Selectable.appearance selectable - Filled.click.chromium.png|16921| Changed|
|Card Converged - Selectable.appearance selectable - Outline - Dark Mode.click.chromium.png|8475| Changed|
|Card Converged - Selectable.appearance selectable - Filled.hover.chromium.png|17294| Changed|
|Card Converged - Selectable.appearance selectable - Filled.selected.chromium.png|17380| Changed|
|Card Converged - Selectable.appearance selectable - Outline - Dark Mode.selected.chromium.png|8554| Changed|
|Card Converged - Selectable.appearance selectable - Outline - Dark Mode.normal.chromium.png|8435| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.selected.chromium.png|17250| Changed|
|Card Converged - Selectable.appearance selectable - Outline - Dark Mode.hover.chromium.png|8554| Changed|
|Card Converged - Selectable.appearance selectable - Outline - High Contrast.selected.chromium.png|12032| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.selected.chromium.png|17244| Changed|
|Card Converged - Selectable.appearance selectable - Outline - High Contrast.click.chromium.png|12032| Changed|
|Card Converged - Selectable.appearance selectable - Filled.normal.chromium.png|17967| Changed|
|Card Converged - Selectable.appearance selectable - Outline - High Contrast.normal.chromium.png|11812| Changed|
|Card Converged - Selectable.appearance selectable - Outline - RTL.selected.chromium.png|18289| Changed|
|Card Converged - Selectable.appearance selectable - Outline - RTL.click.chromium.png|18366| Changed|
|Card Converged - Selectable.appearance selectable - Outline - High Contrast.hover.chromium.png|12032| Changed|
|Card Converged - Selectable.appearance selectable - Outline - RTL.hover.chromium.png|18289| Changed|
|Card Converged - Selectable.appearance selectable - Outline.click.chromium.png|18358| Changed|
|Card Converged - Selectable.appearance selectable - Outline.hover.chromium.png|18281| Changed|
|Card Converged - Selectable.appearance selectable - Outline.selected.chromium.png|18281| Changed|
|Card Converged - Selectable.appearance selectable - Outline.normal.chromium.png|18243| Changed|
|Card Converged - Selectable.appearance selectable - Outline - RTL.normal.chromium.png|18251| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - Dark Mode.click.chromium.png|7878| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.click.chromium.png|12668| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - Dark Mode.selected.chromium.png|8306| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.normal.chromium.png|11812| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - Dark Mode.hover.chromium.png|8142| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - RTL.click.chromium.png|17248| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - Dark Mode.normal.chromium.png|7951| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - RTL.selected.chromium.png|17719| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.hover.chromium.png|12668| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.selected.chromium.png|12032| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - RTL.normal.chromium.png|18075| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - RTL.hover.chromium.png|17530| Changed|
|Card Converged - Selectable.appearance selectable - Subtle.hover.chromium.png|17523| Changed|
|Card Converged - Selectable.appearance selectable - Subtle.normal.chromium.png|18067| Changed|
|Card Converged - Selectable.appearance selectable - Subtle.click.chromium.png|17242| Changed|
|Card Converged - Selectable.appearance selectable - Subtle.selected.chromium.png|17712| Changed|

</details>


  <details><summary><b>Divider Converged - Horizontal</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Divider Converged - Horizontal.Appearance brand - Dark Mode.default.chromium.png|5618| Changed|
|Divider Converged - Horizontal.Appearance brand - High Contrast.default.chromium.png|6694| Changed|
|Divider Converged - Horizontal.Appearance strong - Dark Mode.default.chromium.png|5618| Changed|
|Divider Converged - Horizontal.Appearance strong - High Contrast.default.chromium.png|6694| Changed|
|Divider Converged - Horizontal.Appearance subtle - Dark Mode.default.chromium.png|5618| Changed|
|Divider Converged - Horizontal.with content - High Contrast.default.chromium.png|6694| Changed|
|Divider Converged - Horizontal.Appearance subtle - High Contrast.default.chromium.png|6694| Changed|
|Divider Converged - Horizontal.with content - Dark Mode.default.chromium.png|5618| Changed|

</details>


  <details><summary><b>Divider Converged - Vertical</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Divider Converged - Vertical.Center Aligned.default.chromium.png|5807| Changed|
|Divider Converged - Vertical.inset.default.chromium.png|5807| Changed|
|Divider Converged - Vertical.End Aligned.default.chromium.png|5807| Changed|
|Divider Converged - Vertical.Start Aligned.default.chromium.png|5807| Changed|

</details>


  <details><summary><b>Dropdown Converged</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dropdown Converged.Open with inlinePopup.active option.chromium.png|499| Changed|
|Dropdown Converged.When rendering inline, it should render on top of relatively positioned elements.active option.chromium.png|596| Changed|
|Dropdown Converged.When rendering inline, it should render on top of relatively positioned elements.default.chromium.png|147| Changed|
|Dropdown Converged.When rendering inline, it should render on top of relatively positioned elements.hover.chromium.png|116| Changed|

</details>


  <details><summary><b>Field</b> 30 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Field.Checkbox.default.chromium.png|13746| Changed|
|Field.Input+horizontal.default.chromium.png|13792| Changed|
|Field.Combobox+horizontal.default.chromium.png|13953| Changed|
|Field.Dropdown+horizontal.default.chromium.png|13921| Changed|
|Field.Combobox.default.chromium.png|29410| Changed|
|Field.SpinButton+horizontal.default.chromium.png|11758| Changed|
|Field.ProgressBar.default.chromium.png|23462| Changed|
|Field.RadioGroup+horizontal.default.chromium.png|12165| Changed|
|Field.Dropdown.default.chromium.png|29410| Changed|
|Field.base.default.chromium.png|7709| Changed|
|Field.Slider.default.chromium.png|16044| Changed|
|Field.Switch.default.chromium.png|17133| Changed|
|Field.RadioGroup.default.chromium.png|29086| Changed|
|Field.Select+horizontal.default.chromium.png|14094| Changed|
|Field.horizontal.default.chromium.png|11696| Changed|
|Field.required.default.chromium.png|7709| Changed|
|Field.Textarea+horizontal.default.chromium.png|21140| Changed|
|Field.hint.default.chromium.png|11428| Changed|
|Field.horizontal+longLabel.default.chromium.png|29787| Changed|
|Field.size-small.default.chromium.png|6514| Changed|
|Field.size-large.default.chromium.png|8705| Changed|
|Field.validation-error.default.chromium.png|9471| Changed|
|Field.Select.default.chromium.png|29410| Changed|
|Field.validation-warning.default.chromium.png|9502| Changed|
|Field.validation-none.default.chromium.png|9502| Changed|
|Field.validationMessageIcon.default.chromium.png|9471| Changed|
|Field.horizontal+noLabel.default.chromium.png|22736| Changed|
|Field.validation-success.default.chromium.png|20703| Changed|
|Field.Textarea.default.chromium.png|38973| Changed|
|Field.SpinButton.default.chromium.png|29410| Changed|

</details>


  <details><summary><b>InfoLabel</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|InfoLabel.default - Dark Mode.rest.chromium.png|5684| Changed|
|InfoLabel.default - High Contrast.rest.chromium.png|6773| Changed|

</details>


  <details><summary><b>Label Converged</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Label Converged.Custom Required - RTL.default.chromium.png|0| Added|
|Label Converged.Custom Required.default.chromium.png|0| Added|
|Label Converged.CustomRequired - RTL.default.chromium.png|0| Removed|
|Label Converged.CustomRequired.default.chromium.png|0| Removed|

</details>


  <details><summary><b>Link Converged - Rendered as button</b> 16 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Link Converged - Rendered as button.Stand-alone - Dark Mode.default.chromium.png|0| Added|
|Link Converged - Rendered as button.Stand-alone - Dark Mode.focused.chromium.png|0| Added|
|Link Converged - Rendered as button.Stand-alone - Dark Mode.hover.chromium.png|0| Added|
|Link Converged - Rendered as button.Stand-alone - Dark Mode.pressed.chromium.png|0| Added|
|Link Converged - Rendered as button.Stand-alone - High Contrast.default.chromium.png|0| Added|
|Link Converged - Rendered as button.Stand-alone - High Contrast.focused.chromium.png|0| Added|
|Link Converged - Rendered as button.Stand-alone - High Contrast.hover.chromium.png|0| Added|
|Link Converged - Rendered as button.Stand-alone - High Contrast.pressed.chromium.png|0| Added|
|Link Converged - Rendered as button.Stand-alone Disabled Focusable - Dark Mode.default.chromium.png|0| Added|
|Link Converged - Rendered as button.Stand-alone Disabled Focusable - Dark Mode.focused.chromium.png|0| Added|
|Link Converged - Rendered as button.Stand-alone Disabled Focusable - Dark Mode.hover.chromium.png|0| Added|
|Link Converged - Rendered as button.Stand-alone Disabled Focusable - Dark Mode.pressed.chromium.png|0| Added|
|Link Converged - Rendered as button.Stand-alone Disabled Focusable - High Contrast.default.chromium.png|0| Added|
|Link Converged - Rendered as button.Stand-alone Disabled Focusable - High Contrast.focused.chromium.png|0| Added|
|Link Converged - Rendered as button.Stand-alone Disabled Focusable - High Contrast.hover.chromium.png|0| Added|
|Link Converged - Rendered as button.Stand-alone Disabled Focusable - High Contrast.pressed.chromium.png|0| Added|

</details>


  <details><summary><b>Link Converged - Rendered as span</b> 26 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Link Converged - Rendered as span.Inline Disabled.default.chromium.png|0| Added|
|Link Converged - Rendered as span.Inline Disabled.hover.chromium.png|0| Added|
|Link Converged - Rendered as span.Inline Disabled.pressed.chromium.png|0| Added|
|Link Converged - Rendered as span.Stand-alone - Dark Mode.default.chromium.png|0| Added|
|Link Converged - Rendered as span.Stand-alone - Dark Mode.focused.chromium.png|0| Added|
|Link Converged - Rendered as span.Stand-alone - Dark Mode.hover.chromium.png|0| Added|
|Link Converged - Rendered as span.Stand-alone - Dark Mode.pressed.chromium.png|0| Added|
|Link Converged - Rendered as span.Stand-alone - High Contrast.default.chromium.png|0| Added|
|Link Converged - Rendered as span.Stand-alone - High Contrast.focused.chromium.png|0| Added|
|Link Converged - Rendered as span.Stand-alone - High Contrast.hover.chromium.png|0| Added|
|Link Converged - Rendered as span.Stand-alone - High Contrast.pressed.chromium.png|0| Added|
|Link Converged - Rendered as span.Stand-alone Disabled Focusable - Dark Mode.default.chromium.png|0| Added|
|Link Converged - Rendered as span.Stand-alone Disabled Focusable - Dark Mode.focused.chromium.png|0| Added|
|Link Converged - Rendered as span.Stand-alone Disabled Focusable - Dark Mode.hover.chromium.png|0| Added|
|Link Converged - Rendered as span.Stand-alone Disabled Focusable - Dark Mode.pressed.chromium.png|0| Added|
|Link Converged - Rendered as span.Stand-alone Disabled Focusable - High Contrast.default.chromium.png|0| Added|
|Link Converged - Rendered as span.Stand-alone Disabled Focusable - High Contrast.focused.chromium.png|0| Added|
|Link Converged - Rendered as span.Stand-alone Disabled Focusable - High Contrast.hover.chromium.png|0| Added|
|Link Converged - Rendered as span.Stand-alone Disabled Focusable - High Contrast.pressed.chromium.png|0| Added|
|Link Converged - Rendered as span.Stand-alone Disabled.default.chromium.png|0| Added|
|Link Converged - Rendered as span.Stand-alone Disabled.hover.chromium.png|0| Added|
|Link Converged - Rendered as span.Stand-alone Disabled.pressed.chromium.png|0| Added|
|Link Converged - Rendered as span.Wraps correctly as an inline element.default.chromium.png|0| Removed|
|Link Converged - Rendered as span.Wraps correctly as an inline element.focused.chromium.png|0| Removed|
|Link Converged - Rendered as span.Wraps correctly as an inline element.hover.chromium.png|0| Removed|
|Link Converged - Rendered as span.Wraps correctly as an inline element.pressed.chromium.png|0| Removed|

</details>


  <details><summary><b>MakeStyles</b> 7 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|MakeStyles.RTL- two components in a single Provider.normal.chromium.png|13590| Changed|
|MakeStyles.RTL- two providers with different directions as children.normal.chromium.png|13822| Changed|
|MakeStyles.RTL- two providers with different directions as siblings.normal.chromium.png|18782| Changed|
|MakeStyles.insertion is ordered.boxA, focus+hover.chromium.png|0| Added|
|MakeStyles.insertion is ordered.boxA, focus.chromium.png|0| Added|
|MakeStyles.insertion is ordered.boxB, focus+hover.chromium.png|0| Added|
|MakeStyles.insertion is ordered.normal.chromium.png|0| Added|

</details>


  <details><summary><b>Menu Converged</b> 26 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged.Groups - Dark Mode.chromium.png|0| Added|
|Menu Converged.Groups - Dark Mode.hover menuitem.chromium.png|0| Added|
|Menu Converged.Groups - High Contrast.chromium.png|0| Added|
|Menu Converged.Groups - High Contrast.hover menuitem.chromium.png|0| Added|
|Menu Converged.Groups - RTL.chromium.png|0| Added|
|Menu Converged.Groups - RTL.hover menuitem.chromium.png|0| Added|
|Menu Converged.Groups.chromium.png|0| Added|
|Menu Converged.Groups.hover menuitem.chromium.png|0| Added|
|Menu Converged.Icon Slotted Content - RTL.chromium.png|0| Added|
|Menu Converged.Icon Slotted Content - RTL.hover menuitem.chromium.png|0| Added|
|Menu Converged.Icon Slotted Content.chromium.png|0| Added|
|Menu Converged.Icon Slotted Content.hover menuitem.chromium.png|0| Added|
|Menu Converged.Long Content - RTL.chromium.png|0| Added|
|Menu Converged.Long Content - RTL.hover menuitem.chromium.png|0| Added|
|Menu Converged.Long Content.chromium.png|0| Added|
|Menu Converged.Long Content.hover menuitem.chromium.png|0| Added|
|Menu Converged.Menu Item Links.chromium.png|0| Added|
|Menu Converged.Menu Item Links.hover menuitemlink.chromium.png|0| Added|
|Menu Converged.Secondary Content - RTL.chromium.png|0| Added|
|Menu Converged.Secondary Content - RTL.hover menuitem.chromium.png|0| Added|
|Menu Converged.Secondary Content.chromium.png|0| Added|
|Menu Converged.Secondary Content.hover menuitem.chromium.png|0| Added|
|Menu Converged.default - RTL.chromium.png|0| Added|
|Menu Converged.default - RTL.hover menuitem.chromium.png|0| Added|
|Menu Converged.default.chromium.png|0| Added|
|Menu Converged.default.hover menuitem.chromium.png|0| Added|

</details>


  <details><summary><b>Menu Converged - nested menu</b> 14 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - nested menu.Nested Within Menu Trigger.chromium.png|0| Added|
|Menu Converged - nested menu.Nested Within Menu Trigger.submenu open.chromium.png|0| Added|
|Menu Converged - nested menu.nested submenus - RTL.all open.chromium.png|0| Added|
|Menu Converged - nested menu.nested submenus - RTL.chromium.png|0| Added|
|Menu Converged - nested menu.nested submenus.all open.chromium.png|0| Added|
|Menu Converged - nested menu.nested submenus.chromium.png|0| Added|
|Menu Converged - nested menu.split item - RTL.all open.chromium.png|0| Added|
|Menu Converged - nested menu.split item - RTL.chromium.png|0| Added|
|Menu Converged - nested menu.split item.all open.chromium.png|0| Added|
|Menu Converged - nested menu.split item.chromium.png|0| Added|
|Menu Converged - nested menu.submenuIndicator slotted content - RTL.chromium.png|0| Added|
|Menu Converged - nested menu.submenuIndicator slotted content - RTL.submenus open.chromium.png|0| Added|
|Menu Converged - nested menu.submenuIndicator slotted content.chromium.png|0| Added|
|Menu Converged - nested menu.submenuIndicator slotted content.submenus open.chromium.png|0| Added|

</details>


  <details><summary><b>Menu Converged - selection</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - selection.groups - RTL.chromium.png|0| Added|
|Menu Converged - selection.groups - RTL.selected.chromium.png|0| Added|
|Menu Converged - selection.groups.chromium.png|0| Added|
|Menu Converged - selection.groups.selected.chromium.png|0| Added|

</details>


  <details><summary><b>Persona Converged</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Persona Converged.basic - Dark Mode.normal.chromium.png|15896| Changed|
|Persona Converged.basic - High Contrast.normal.chromium.png|3128| Changed|
|Persona Converged.basic - RTL.normal.chromium.png|86271| Changed|
|Persona Converged.textPosition.normal.chromium.png|42095| Changed|
|Persona Converged.basic.normal.chromium.png|88626| Changed|
|Persona Converged.textAlignment.normal.chromium.png|62023| Changed|
|Persona Converged.size-presence.normal.chromium.png|280778| Changed|
|Persona Converged.textWrap.normal.chromium.png|131006| Changed|
|Persona Converged.size-avatar.normal.chromium.png|265964| Changed|

</details>


  <details><summary><b>Popover Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Popover Converged.positioning - High Contrast.positioned popovers.chromium.png|24384| Changed|
|Popover Converged.positioning - High Contrast.chromium.png|638978| Changed|
|Popover Converged.positioning - RTL.chromium.png|1476| Changed|

</details>


  <details><summary><b>Positioning</b> 30 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Positioning.arrow - RTL.chromium.png|13876| Changed|
|Positioning.Target display none.target display- none.chromium.png|186| Changed|
|Positioning.auto size overflow padding - RTL.chromium.png|5180| Changed|
|Positioning.Multiple scroll parents.chromium.png|12292| Changed|
|Positioning.Multiple scroll parents.container attached to target.chromium.png|16379| Changed|
|Positioning.disable CSS transform with position fixed - RTL.chromium.png|7099| Changed|
|Positioning.coverTarget - RTL.chromium.png|5100| Changed|
|Positioning.disable CSS transform - RTL.chromium.png|7099| Changed|
|Positioning.horizontal flip - RTL.chromium.png|2563| Changed|
|Positioning.disable tether.chromium.png|4614| Changed|
|Positioning.explicit overflow padding - RTL.chromium.png|2979| Changed|
|Positioning.horizontal overflow - RTL.chromium.png|2228| Changed|
|Positioning.horizontal overflow padding - RTL.chromium.png|2595| Changed|
|Positioning.offset function - RTL.chromium.png|7404| Changed|
|Positioning.position fixed - RTL.chromium.png|7099| Changed|
|Positioning.offset - RTL.chromium.png|7161| Changed|
|Positioning.position and align props - RTL.chromium.png|7099| Changed|
|Positioning.visibility modifiers.has  [data-popper-escaped]  when the popper escapes the reference element's boundary.chromium.png|24159| Changed|
|Positioning.visibility modifiers.has  [data-popper-is-intersecting]  when the popover intersects boundaries.chromium.png|32855| Changed|
|Positioning.visibility modifiers.has  [data-popper-reference-hidden]  when the reference is hidden.chromium.png|29955| Changed|
|Positioning.Cover Target.chromium.png|0| Added|
|Positioning.Explicit Overflow Padding.chromium.png|0| Added|
|Positioning.Horizontal Flip.chromium.png|0| Added|
|Positioning.Horizontal Overflow Padding.chromium.png|0| Added|
|Positioning.Offset Function.chromium.png|0| Added|
|Positioning.coverTarget.chromium.png|0| Removed|
|Positioning.explicit overflow padding.chromium.png|0| Removed|
|Positioning.horizontal flip.chromium.png|0| Removed|
|Positioning.horizontal overflow padding.chromium.png|0| Removed|
|Positioning.offset function.chromium.png|0| Removed|

</details>


  <details><summary><b>Positioning (no decorator)</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Positioning (no decorator).scroll jumps.positions without scroll jump.chromium.png|0| Removed|

</details>


  <details><summary><b>PresenceBadge Converged</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|PresenceBadge Converged.Inverted Background - Dark Mode.default.chromium.png|0| Added|
|PresenceBadge Converged.Inverted Background - High Contrast.default.chromium.png|0| Added|
|PresenceBadge Converged.Inverted Background.default.chromium.png|0| Added|
|PresenceBadge Converged.Out Of Office - RTL.default.chromium.png|0| Added|
|PresenceBadge Converged.Out Of Office.default.chromium.png|0| Added|
|PresenceBadge Converged.Sizes - RTL.default.chromium.png|0| Added|
|PresenceBadge Converged.Sizes.default.chromium.png|0| Added|
|PresenceBadge Converged.Status - RTL.default.chromium.png|0| Added|
|PresenceBadge Converged.Status.default.chromium.png|0| Added|

</details>


  <details><summary><b>ProgressBar converged</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ProgressBar converged.Determinate + thickness - Dark Mode.default.chromium.png|5785| Changed|
|ProgressBar converged.Determinate + thickness - High Contrast.default.chromium.png|6893| Changed|
|ProgressBar converged.Indeterminate + thickness - Dark Mode.chromium.png|66694| Changed|
|ProgressBar converged.Indeterminate + thickness - High Contrast.chromium.png|79472| Changed|
|ProgressBar converged.Indeterminate + thickness - RTL.chromium.png|677| Changed|

</details>


  <details><summary><b>Radio Converged</b> 20 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Radio Converged.checked - Dark Mode.hover.chromium.png|5885| Changed|
|Radio Converged.checked - Dark Mode.active.chromium.png|5885| Changed|
|Radio Converged.checked - Dark Mode.rest.chromium.png|5885| Changed|
|Radio Converged.checked - High Contrast.active.chromium.png|7013| Changed|
|Radio Converged.checked - High Contrast.hover.chromium.png|7013| Changed|
|Radio Converged.checked - High Contrast.rest.chromium.png|7013| Changed|
|Radio Converged.disabled - Dark Mode.active.chromium.png|5885| Changed|
|Radio Converged.disabled - Dark Mode.hover.chromium.png|5885| Changed|
|Radio Converged.disabled - Dark Mode.rest.chromium.png|5885| Changed|
|Radio Converged.disabled - High Contrast.active.chromium.png|7013| Changed|
|Radio Converged.disabled - High Contrast.hover.chromium.png|7013| Changed|
|Radio Converged.disabled - High Contrast.rest.chromium.png|7013| Changed|
|Radio Converged.disabled+checked - Dark Mode.default.chromium.png|5885| Changed|
|Radio Converged.disabled+checked - High Contrast.default.chromium.png|7013| Changed|
|Radio Converged.unchecked - Dark Mode.active.chromium.png|5885| Changed|
|Radio Converged.unchecked - Dark Mode.hover.chromium.png|5885| Changed|
|Radio Converged.unchecked - Dark Mode.rest.chromium.png|5885| Changed|
|Radio Converged.unchecked - High Contrast.hover.chromium.png|7013| Changed|
|Radio Converged.unchecked - High Contrast.active.chromium.png|7013| Changed|
|Radio Converged.unchecked - High Contrast.rest.chromium.png|7013| Changed|

</details>


  <details><summary><b>Rating Converged</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Rating Converged.Brand Rating with half star - Dark Mode.hover.chromium.png|5818| Changed|
|Rating Converged.Brand Rating with half star - Dark Mode.rest.chromium.png|5818| Changed|
|Rating Converged.Brand Rating with half star - High Contrast.rest.chromium.png|6933| Changed|
|Rating Converged.Brand Rating with half star - High Contrast.hover.chromium.png|6933| Changed|
|Rating Converged.Marigold Rating with half star - Dark Mode.hover.chromium.png|5818| Changed|
|Rating Converged.Marigold Rating with half star - Dark Mode.rest.chromium.png|5818| Changed|
|Rating Converged.Marigold Rating with half star - High Contrast.hover.chromium.png|6933| Changed|
|Rating Converged.Neutral Rating with half star - Dark Mode.hover.chromium.png|5818| Changed|
|Rating Converged.Neutral Rating with half star - Dark Mode.rest.chromium.png|5818| Changed|
|Rating Converged.Neutral Rating with half star - High Contrast.hover.chromium.png|6933| Changed|
|Rating Converged.Marigold Rating with half star - High Contrast.rest.chromium.png|6933| Changed|
|Rating Converged.Neutral Rating with half star - High Contrast.rest.chromium.png|6933| Changed|

</details>


  <details><summary><b>RatingDisplay Converged</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|RatingDisplay Converged.Brand with half value - High Contrast.rest.chromium.png|6694| Changed|
|RatingDisplay Converged.Marigold with half value - High Contrast.rest.chromium.png|6694| Changed|
|RatingDisplay Converged.Brand with half value - Dark Mode.rest.chromium.png|5618| Changed|
|RatingDisplay Converged.Marigold with half value - Dark Mode.rest.chromium.png|5618| Changed|
|RatingDisplay Converged.Neutral with half value - High Contrast.rest.chromium.png|6694| Changed|
|RatingDisplay Converged.Neutral with half value - Dark Mode.rest.chromium.png|5618| Changed|

</details>


  <details><summary><b>Shadow DOM</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Shadow DOM.Default.chromium.png|0| Added|
|Shadow DOM.Default.default.chromium.png|0| Removed|
|Shadow DOM.Portal.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Skeleton converged</b> 15 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Skeleton converged.Opaque Skeleton with circle - Dark Mode.chromium.png|53286| Changed|
|Skeleton converged.Opaque Skeleton with rectangle - High Contrast.chromium.png|63495| Changed|
|Skeleton converged.Opaque Skeleton with rectangle - RTL.chromium.png|66| Changed|
|Skeleton converged.Opaque Skeleton with rectangle - Dark Mode.chromium.png|53286| Changed|
|Skeleton converged.Opaque Skeleton with circle - RTL.chromium.png|8| Changed|
|Skeleton converged.Opaque Skeleton with circle - High Contrast.chromium.png|63495| Changed|
|Skeleton converged.Opaque Skeleton with square - Dark Mode.chromium.png|53286| Changed|
|Skeleton converged.Opaque Skeleton with square - High Contrast.chromium.png|63495| Changed|
|Skeleton converged.Opaque Skeleton with square - RTL.chromium.png|11| Changed|
|Skeleton converged.Translucent Skeleton with circle - High Contrast.chromium.png|63495| Changed|
|Skeleton converged.Translucent Skeleton with circle - Dark Mode.chromium.png|53286| Changed|
|Skeleton converged.Translucent Skeleton with rectangle - High Contrast.chromium.png|63495| Changed|
|Skeleton converged.Translucent Skeleton with rectangle - Dark Mode.chromium.png|53286| Changed|
|Skeleton converged.Translucent Skeleton with square - Dark Mode.chromium.png|53286| Changed|
|Skeleton converged.Translucent Skeleton with square - High Contrast.chromium.png|63495| Changed|

</details>


  <details><summary><b>Slider Converged</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Slider Converged.Disabled - Dark Mode.default.chromium.png|3026| Changed|
|Slider Converged.Disabled - Dark Mode.hover.chromium.png|3026| Changed|
|Slider Converged.Disabled - Dark Mode.pressed.chromium.png|3026| Changed|
|Slider Converged.Disabled - High Contrast.default.chromium.png|3606| Changed|
|Slider Converged.Disabled - High Contrast.hover.chromium.png|3606| Changed|
|Slider Converged.Disabled - High Contrast.pressed.chromium.png|3606| Changed|
|Slider Converged.Root - Dark Mode.hover.chromium.png|3026| Changed|
|Slider Converged.Root - Dark Mode.default.chromium.png|3026| Changed|
|Slider Converged.Root - Dark Mode.pressed.chromium.png|3026| Changed|
|Slider Converged.Root - High Contrast.hover.chromium.png|3606| Changed|
|Slider Converged.Root - High Contrast.pressed.chromium.png|3606| Changed|
|Slider Converged.Root - High Contrast.default.chromium.png|3606| Changed|

</details>


  <details><summary><b>SpinButton Converged</b> 117 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.hoverDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.focused.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.hoverIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.hoverInput.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.mouseDownIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.mouseDownDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.hoverIncrement.chromium.png|7013| Changed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.rest.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.hoverDecrement.chromium.png|7013| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.hoverInput.chromium.png|7013| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.mouseDownIncrement.chromium.png|7013| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.focused.chromium.png|7013| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.mouseDownDecrement.chromium.png|7013| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.rest.chromium.png|7013| Changed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.hoverIncrement.chromium.png|6219| Changed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.hoverInput.chromium.png|6219| Changed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.focused.chromium.png|6219| Changed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.hoverDecrement.chromium.png|6219| Changed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.mouseDownDecrement.chromium.png|6219| Changed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.mouseDownIncrement.chromium.png|6219| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.mouseDownDecrement.chromium.png|7411| Changed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.rest.chromium.png|6219| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.focused.chromium.png|7411| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.hoverDecrement.chromium.png|7411| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.hoverIncrement.chromium.png|7411| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.hoverInput.chromium.png|7411| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.mouseDownIncrement.chromium.png|7411| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.rest.chromium.png|7411| Changed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.focused.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.hoverDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.hoverIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.rest.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.focused.chromium.png|7013| Changed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.mouseDownDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.mouseDownIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.mouseDownDecrement.chromium.png|7013| Changed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.hoverIncrement.chromium.png|7013| Changed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.hoverInput.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.hoverDecrement.chromium.png|7013| Changed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.rest.chromium.png|7013| Changed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.mouseDownIncrement.chromium.png|7013| Changed|
|SpinButton Converged.Appearance- underline - Dark Mode.focused.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.focused.chromium.png|7013| Changed|
|SpinButton Converged.Appearance- underline - Dark Mode.mouseDownIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.hoverIncrement.chromium.png|7013| Changed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.hoverInput.chromium.png|7013| Changed|
|SpinButton Converged.Appearance- underline - Dark Mode.mouseDownDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.hoverDecrement.chromium.png|7013| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.mouseDownIncrement.chromium.png|7013| Changed|
|SpinButton Converged.Appearance- underline - Dark Mode.hoverIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- underline - Dark Mode.hoverInput.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- underline - Dark Mode.hoverDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- underline - Dark Mode.rest.chromium.png|5885| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.mouseDownDecrement.chromium.png|7013| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.hoverInput.chromium.png|7013| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.rest.chromium.png|7013| Changed|
|SpinButton Converged.At Max Bound - Dark Mode.mouseDownIncrement.chromium.png|5885| Changed|
|SpinButton Converged.At Max Bound - Dark Mode.focused.chromium.png|5885| Changed|
|SpinButton Converged.At Max Bound - Dark Mode.hoverDecrement.chromium.png|5885| Changed|
|SpinButton Converged.At Max Bound - Dark Mode.hoverIncrement.chromium.png|5885| Changed|
|SpinButton Converged.At Max Bound - High Contrast.hoverDecrement.chromium.png|7013| Changed|
|SpinButton Converged.At Max Bound - Dark Mode.rest.chromium.png|5885| Changed|
|SpinButton Converged.At Max Bound - Dark Mode.hoverInput.chromium.png|5885| Changed|
|SpinButton Converged.At Max Bound - Dark Mode.mouseDownDecrement.chromium.png|5885| Changed|
|SpinButton Converged.At Max Bound - High Contrast.focused.chromium.png|7013| Changed|
|SpinButton Converged.At Max Bound - High Contrast.hoverInput.chromium.png|7013| Changed|
|SpinButton Converged.At Max Bound - High Contrast.hoverIncrement.chromium.png|7013| Changed|
|SpinButton Converged.At Max Bound - High Contrast.mouseDownDecrement.chromium.png|7013| Changed|
|SpinButton Converged.At Min Bound - Dark Mode.mouseDownDecrement.chromium.png|5885| Changed|
|SpinButton Converged.At Min Bound - Dark Mode.hoverDecrement.chromium.png|5885| Changed|
|SpinButton Converged.At Min Bound - Dark Mode.focused.chromium.png|5885| Changed|
|SpinButton Converged.At Min Bound - High Contrast.hoverDecrement.chromium.png|7013| Changed|
|SpinButton Converged.At Max Bound - High Contrast.rest.chromium.png|7013| Changed|
|SpinButton Converged.At Min Bound - Dark Mode.rest.chromium.png|5885| Changed|
|SpinButton Converged.At Min Bound - Dark Mode.mouseDownIncrement.chromium.png|5885| Changed|
|SpinButton Converged.At Min Bound - Dark Mode.hoverIncrement.chromium.png|5885| Changed|
|SpinButton Converged.At Min Bound - High Contrast.focused.chromium.png|7013| Changed|
|SpinButton Converged.At Min Bound - High Contrast.hoverIncrement.chromium.png|7013| Changed|
|SpinButton Converged.At Min Bound - High Contrast.mouseDownDecrement.chromium.png|7013| Changed|
|SpinButton Converged.At Max Bound - High Contrast.mouseDownIncrement.chromium.png|7013| Changed|
|SpinButton Converged.At Min Bound - High Contrast.hoverInput.chromium.png|7013| Changed|
|SpinButton Converged.At Min Bound - Dark Mode.hoverInput.chromium.png|5885| Changed|
|SpinButton Converged.At Min Bound - High Contrast.mouseDownIncrement.chromium.png|7013| Changed|
|SpinButton Converged.At Min Bound - High Contrast.rest.chromium.png|7013| Changed|
|SpinButton Converged.Disabled - Dark Mode.hoverDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Disabled - Dark Mode.hoverInput.chromium.png|5885| Changed|
|SpinButton Converged.Disabled - Dark Mode.hoverIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Disabled - Dark Mode.focused.chromium.png|5885| Changed|
|SpinButton Converged.Disabled - High Contrast.hoverDecrement.chromium.png|7013| Changed|
|SpinButton Converged.Disabled - Dark Mode.mouseDownDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Disabled - High Contrast.hoverIncrement.chromium.png|7013| Changed|
|SpinButton Converged.Disabled - Dark Mode.rest.chromium.png|5885| Changed|
|SpinButton Converged.Disabled - High Contrast.focused.chromium.png|7013| Changed|
|SpinButton Converged.Disabled - Dark Mode.mouseDownIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Disabled - High Contrast.mouseDownDecrement.chromium.png|7013| Changed|
|SpinButton Converged.Disabled - High Contrast.rest.chromium.png|7013| Changed|
|SpinButton Converged.Disabled - High Contrast.hoverInput.chromium.png|7013| Changed|
|SpinButton Converged.Disabled - High Contrast.mouseDownIncrement.chromium.png|7013| Changed|
|SpinButton Converged.Display Value - Dark Mode.focused.chromium.png|5885| Changed|
|SpinButton Converged.Display Value - Dark Mode.hoverDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Display Value - Dark Mode.hoverIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Display Value - High Contrast.hoverIncrement.chromium.png|7013| Changed|
|SpinButton Converged.Display Value - High Contrast.hoverDecrement.chromium.png|7013| Changed|
|SpinButton Converged.Display Value - Dark Mode.rest.chromium.png|5885| Changed|
|SpinButton Converged.Display Value - Dark Mode.mouseDownIncrement.chromium.png|5885| Changed|
|SpinButton Converged.Display Value - Dark Mode.mouseDownDecrement.chromium.png|5885| Changed|
|SpinButton Converged.Display Value - High Contrast.mouseDownIncrement.chromium.png|7013| Changed|
|SpinButton Converged.Display Value - High Contrast.rest.chromium.png|7013| Changed|
|SpinButton Converged.Display Value - Dark Mode.hoverInput.chromium.png|5885| Changed|
|SpinButton Converged.Display Value - High Contrast.mouseDownDecrement.chromium.png|7013| Changed|
|SpinButton Converged.Display Value - High Contrast.focused.chromium.png|7013| Changed|
|SpinButton Converged.Display Value - High Contrast.hoverInput.chromium.png|7013| Changed|
|SpinButton Converged.Size- small - High Contrast.default.chromium.png|6853| Changed|
|SpinButton Converged.Size- small - Dark Mode.default.chromium.png|5751| Changed|
|SpinButton Converged.Custom Width - Dark Mode.chromium.png|74740| Changed|
|SpinButton Converged.Custom Width - RTL.chromium.png|220| Changed|
|SpinButton Converged.Custom Width - High Contrast.chromium.png|89059| Changed|

</details>


  <details><summary><b>Spinner converged</b> 10 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Spinner converged.Primary - Dark Mode.chromium.png|74740| Changed|
|Spinner converged.Inverted - High Contrast.chromium.png|121013| Changed|
|Spinner converged.Inverted with Label - High Contrast.chromium.png|466124| Changed|
|Spinner converged.Inverted - Dark Mode.chromium.png|101556| Changed|
|Spinner converged.Inverted with Label - Dark Mode.chromium.png|391179| Changed|
|Spinner converged.Primary - High Contrast.chromium.png|89059| Changed|
|Spinner converged.Animation - RTL.chromium.png|14230| Changed|
|Spinner converged.Primary with Label - Dark Mode.chromium.png|364362| Changed|
|Spinner converged.Primary with Label - RTL.chromium.png|2493| Changed|
|Spinner converged.Primary with Label - High Contrast.chromium.png|434169| Changed|

</details>


  <details><summary><b>Switch Converged</b> 39 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Switch Converged.Disabled and checked - Dark Mode.default.chromium.png|3524| Changed|
|Switch Converged.Disabled and checked - Dark Mode.hover.chromium.png|3524| Changed|
|Switch Converged.Disabled and checked - Dark Mode.pressed.chromium.png|3524| Changed|
|Switch Converged.Disabled and checked - High Contrast.default.chromium.png|4200| Changed|
|Switch Converged.Disabled and checked - High Contrast.hover.chromium.png|4200| Changed|
|Switch Converged.Disabled and checked - High Contrast.pressed.chromium.png|4200| Changed|
|Switch Converged.Disabled and unchecked - Dark Mode.default.chromium.png|3524| Changed|
|Switch Converged.Disabled and unchecked - Dark Mode.hover.chromium.png|3524| Changed|
|Switch Converged.Disabled and unchecked - Dark Mode.pressed.chromium.png|3524| Changed|
|Switch Converged.Disabled and unchecked - High Contrast.default.chromium.png|4200| Changed|
|Switch Converged.Disabled and unchecked - High Contrast.hover.chromium.png|4200| Changed|
|Switch Converged.Disabled and unchecked - High Contrast.pressed.chromium.png|4200| Changed|
|Switch Converged.Enabled and checked - Dark Mode.default.chromium.png|3524| Changed|
|Switch Converged.Enabled and checked - Dark Mode.hover.chromium.png|3524| Changed|
|Switch Converged.Enabled and checked - High Contrast.default.chromium.png|4200| Changed|
|Switch Converged.Enabled and checked - Dark Mode.pressed.chromium.png|3524| Changed|
|Switch Converged.Enabled and checked - High Contrast.pressed.chromium.png|4200| Changed|
|Switch Converged.Enabled and checked - RTL.default.chromium.png|275| Changed|
|Switch Converged.Enabled and checked - RTL.hover.chromium.png|277| Changed|
|Switch Converged.Enabled and checked - RTL.pressed.chromium.png|280| Changed|
|Switch Converged.Enabled and checked - High Contrast.hover.chromium.png|4200| Changed|
|Switch Converged.Enabled and unchecked - Dark Mode.pressed.chromium.png|3524| Changed|
|Switch Converged.Enabled and unchecked - Dark Mode.hover.chromium.png|3524| Changed|
|Switch Converged.Enabled and unchecked - High Contrast.default.chromium.png|4200| Changed|
|Switch Converged.Enabled and unchecked - Dark Mode.default.chromium.png|3524| Changed|
|Switch Converged.Enabled and unchecked - High Contrast.hover.chromium.png|4200| Changed|
|Switch Converged.Enabled and unchecked - RTL.default.chromium.png|284| Changed|
|Switch Converged.Enabled and unchecked - High Contrast.pressed.chromium.png|4200| Changed|
|Switch Converged.Enabled and unchecked - RTL.pressed.chromium.png|290| Changed|
|Switch Converged.Enabled and unchecked - RTL.hover.chromium.png|287| Changed|
|Switch Converged.With label above - RTL.default.chromium.png|284| Changed|
|Switch Converged.With label above - RTL.hover.chromium.png|287| Changed|
|Switch Converged.With label above - RTL.pressed.chromium.png|290| Changed|
|Switch Converged.With label after - RTL.default.chromium.png|284| Changed|
|Switch Converged.With label after - RTL.hover.chromium.png|287| Changed|
|Switch Converged.With label after - RTL.pressed.chromium.png|290| Changed|
|Switch Converged.With label before - RTL.hover.chromium.png|287| Changed|
|Switch Converged.With label before - RTL.pressed.chromium.png|290| Changed|
|Switch Converged.With label before - RTL.default.chromium.png|284| Changed|

</details>


  <details><summary><b>Table Coverged</b> 89 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table Coverged.Multiselect Checked Brand - Dark Mode.chromium.png|0| Added|
|Table Coverged.Multiselect Checked Brand - High Contrast.chromium.png|0| Added|
|Table Coverged.Multiselect Checked Brand - RTL.chromium.png|0| Added|
|Table Coverged.Multiselect Checked Brand Layout Flex - Dark Mode.chromium.png|0| Added|
|Table Coverged.Multiselect Checked Brand Layout Flex - High Contrast.chromium.png|0| Added|
|Table Coverged.Multiselect Checked Brand Layout Flex - RTL.chromium.png|0| Added|
|Table Coverged.Multiselect Checked Brand Layout Flex.chromium.png|0| Added|
|Table Coverged.Multiselect Checked Brand.chromium.png|0| Added|
|Table Coverged.Multiselect Checked Neutral - Dark Mode.chromium.png|0| Added|
|Table Coverged.Multiselect Checked Neutral - High Contrast.chromium.png|0| Added|
|Table Coverged.Multiselect Checked Neutral - RTL.chromium.png|0| Added|
|Table Coverged.Multiselect Checked Neutral Layout Flex - Dark Mode.chromium.png|0| Added|
|Table Coverged.Multiselect Checked Neutral Layout Flex - High Contrast.chromium.png|0| Added|
|Table Coverged.Multiselect Checked Neutral Layout Flex - RTL.chromium.png|0| Added|
|Table Coverged.Multiselect Checked Neutral Layout Flex.chromium.png|0| Added|
|Table Coverged.Multiselect Checked Neutral.chromium.png|0| Added|
|Table Coverged.Multiselect Layout Flex.chromium.png|0| Added|
|Table Coverged.Multiselect Mixed Brand - Dark Mode.chromium.png|0| Added|
|Table Coverged.Multiselect Mixed Brand - High Contrast.chromium.png|0| Added|
|Table Coverged.Multiselect Mixed Brand - RTL.chromium.png|0| Added|
|Table Coverged.Multiselect Mixed Brand Layout Flex - Dark Mode.chromium.png|0| Added|
|Table Coverged.Multiselect Mixed Brand Layout Flex - High Contrast.chromium.png|0| Added|
|Table Coverged.Multiselect Mixed Brand Layout Flex - RTL.chromium.png|0| Added|
|Table Coverged.Multiselect Mixed Brand Layout Flex.chromium.png|0| Added|
|Table Coverged.Multiselect Mixed Brand.chromium.png|0| Added|
|Table Coverged.Multiselect Mixed Neutral - Dark Mode.chromium.png|0| Added|
|Table Coverged.Multiselect Mixed Neutral - High Contrast.chromium.png|0| Added|
|Table Coverged.Multiselect Mixed Neutral - RTL.chromium.png|0| Added|
|Table Coverged.Multiselect Mixed Neutral Layout Flex - Dark Mode.chromium.png|0| Added|
|Table Coverged.Multiselect Mixed Neutral Layout Flex - High Contrast.chromium.png|0| Added|
|Table Coverged.Multiselect Mixed Neutral Layout Flex - RTL.chromium.png|0| Added|
|Table Coverged.Multiselect Mixed Neutral Layout Flex.chromium.png|0| Added|
|Table Coverged.Multiselect Mixed Neutral.chromium.png|0| Added|
|Table Coverged.Multiselect.chromium.png|0| Added|
|Table Coverged.Primary Cell.chromium.png|0| Added|
|Table Coverged.Primary Layout Flex Cell.chromium.png|0| Added|
|Table Coverged.Single Select Checked Brand - Dark Mode.chromium.png|0| Added|
|Table Coverged.Single Select Checked Brand - High Contrast.chromium.png|0| Added|
|Table Coverged.Single Select Checked Brand - RTL.chromium.png|0| Added|
|Table Coverged.Single Select Checked Brand Layout Flex - Dark Mode.chromium.png|0| Added|
|Table Coverged.Single Select Checked Brand Layout Flex - High Contrast.chromium.png|0| Added|
|Table Coverged.Single Select Checked Brand Layout Flex - RTL.chromium.png|0| Added|
|Table Coverged.Single Select Checked Brand Layout Flex.chromium.png|0| Added|
|Table Coverged.Single Select Checked Brand.chromium.png|0| Added|
|Table Coverged.Single Select Checked Layout Flex.chromium.png|0| Added|
|Table Coverged.Single Select Checked Neutral - Dark Mode.chromium.png|0| Added|
|Table Coverged.Single Select Checked Neutral - High Contrast.chromium.png|0| Added|
|Table Coverged.Single Select Checked Neutral - High Contrast.chromium_1.png|0| Added|
|Table Coverged.Single Select Checked Neutral - RTL.chromium.png|0| Added|
|Table Coverged.Single Select Checked Neutral Layout Flex - Dark Mode.chromium.png|0| Added|
|Table Coverged.Single Select Checked Neutral Layout Flex.chromium.png|0| Added|
|Table Coverged.Single Select Checked Neutral.chromium.png|0| Added|
|Table Coverged.Single Select Checked.chromium.png|0| Added|
|Table Coverged.Single Select Layout Flex.chromium.png|0| Added|
|Table Coverged.Single Select.chromium.png|0| Added|
|Table Coverged.Size Extra Layout Flex Small.chromium.png|0| Added|
|Table Coverged.Size Extra Small.chromium.png|0| Added|
|Table Coverged.Size Medium Layout Flex.chromium.png|0| Added|
|Table Coverged.Size Medium.chromium.png|0| Added|
|Table Coverged.Size Small Layout Flex.chromium.png|0| Added|
|Table Coverged.Size Small.chromium.png|0| Added|
|Table Coverged.Subtle Selection Layout Flex.chromium.png|0| Added|
|Table Coverged.Subtle Selection Layout Flex.hover unselected row.chromium.png|0| Added|
|Table Coverged.Subtle Selection.chromium.png|0| Added|
|Table Coverged.Subtle Selection.hover unselected row.chromium.png|0| Added|
|Table Coverged._Multiselect - Dark Mode.chromium.png|0| Added|
|Table Coverged._Multiselect - High Contrast.chromium.png|0| Added|
|Table Coverged._Multiselect - RTL.chromium.png|0| Added|
|Table Coverged._Multiselect Layout Flex - Dark Mode.chromium.png|0| Added|
|Table Coverged._Multiselect Layout Flex - High Contrast.chromium.png|0| Added|
|Table Coverged._Multiselect Layout Flex - RTL.chromium.png|0| Added|
|Table Coverged._Single Select - Dark Mode.chromium.png|0| Added|
|Table Coverged._Single Select - High Contrast.chromium.png|0| Added|
|Table Coverged._Single Select - RTL.chromium.png|0| Added|
|Table Coverged._Single Select Checked - Dark Mode.chromium.png|0| Added|
|Table Coverged._Single Select Checked - High Contrast.chromium.png|0| Added|
|Table Coverged._Single Select Checked - RTL.chromium.png|0| Added|
|Table Coverged._Single Select Checked Layout Flex - Dark Mode.chromium.png|0| Added|
|Table Coverged._Single Select Checked Layout Flex - High Contrast.chromium.png|0| Added|
|Table Coverged._Single Select Checked Layout Flex - RTL.chromium.png|0| Added|
|Table Coverged._Single Select Layout Flex - Dark Mode.chromium.png|0| Added|
|Table Coverged._Single Select Layout Flex - High Contrast.chromium.png|0| Added|
|Table Coverged._Single Select Layout Flex - RTL.chromium.png|0| Added|
|Table Coverged._Size Medium - Dark Mode.chromium.png|0| Added|
|Table Coverged._Size Medium - High Contrast.chromium.png|0| Added|
|Table Coverged._Size Medium - RTL.chromium.png|0| Added|
|Table Coverged._Size Medium Layout Flex - Dark Mode.chromium.png|0| Added|
|Table Coverged._Size Medium Layout Flex - High Contrast.chromium.png|0| Added|
|Table Coverged._Size Medium Layout Flex - RTL.chromium.png|0| Added|

</details>


  <details><summary><b>Table Coverged - cell actions</b> 36 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table Coverged - cell actions.Always Visible - Dark Mode.chromium.png|0| Added|
|Table Coverged - cell actions.Always Visible - Dark Mode.hover row.chromium.png|0| Added|
|Table Coverged - cell actions.Always Visible - High Contrast.chromium.png|0| Added|
|Table Coverged - cell actions.Always Visible - High Contrast.hover row.chromium.png|0| Added|
|Table Coverged - cell actions.Always Visible - RTL.chromium.png|0| Added|
|Table Coverged - cell actions.Always Visible - RTL.hover row.chromium.png|0| Added|
|Table Coverged - cell actions.Always Visible.chromium.png|0| Added|
|Table Coverged - cell actions.Always Visible.hover row.chromium.png|0| Added|
|Table Coverged - cell actions.Default - Dark Mode.chromium.png|0| Added|
|Table Coverged - cell actions.Default - Dark Mode.hover row.chromium.png|0| Added|
|Table Coverged - cell actions.Default - High Contrast.chromium.png|0| Added|
|Table Coverged - cell actions.Default - High Contrast.hover row.chromium.png|0| Added|
|Table Coverged - cell actions.Default - RTL.chromium.png|0| Added|
|Table Coverged - cell actions.Default - RTL.hover row.chromium.png|0| Added|
|Table Coverged - cell actions.Default.chromium.png|0| Added|
|Table Coverged - cell actions.Default.hover row.chromium.png|0| Added|
|Table Coverged - cell actions.Flex Layout - Dark Mode.chromium.png|0| Added|
|Table Coverged - cell actions.Flex Layout - Dark Mode.hover row.chromium.png|0| Added|
|Table Coverged - cell actions.Flex Layout - High Contrast.chromium.png|0| Added|
|Table Coverged - cell actions.Flex Layout - High Contrast.hover row.chromium.png|0| Added|
|Table Coverged - cell actions.Flex Layout - RTL.chromium.png|0| Added|
|Table Coverged - cell actions.Flex Layout - RTL.hover row.chromium.png|0| Added|
|Table Coverged - cell actions.Flex Layout Always Visible - Dark Mode.chromium.png|0| Added|
|Table Coverged - cell actions.Flex Layout Always Visible - Dark Mode.hover row.chromium.png|0| Added|
|Table Coverged - cell actions.Flex Layout Always Visible - High Contrast.chromium.png|0| Added|
|Table Coverged - cell actions.Flex Layout Always Visible - High Contrast.hover row.chromium.png|0| Added|
|Table Coverged - cell actions.Flex Layout Always Visible - RTL.chromium.png|0| Added|
|Table Coverged - cell actions.Flex Layout Always Visible - RTL.hover row.chromium.png|0| Added|
|Table Coverged - cell actions.Flex Layout Always Visible.chromium.png|0| Added|
|Table Coverged - cell actions.Flex Layout Always Visible.hover row.chromium.png|0| Added|
|Table Coverged - cell actions.Flex Layout In Header Cell.chromium.png|0| Added|
|Table Coverged - cell actions.Flex Layout In Header Cell.hover row.chromium.png|0| Added|
|Table Coverged - cell actions.Flex Layout.chromium.png|0| Added|
|Table Coverged - cell actions.Flex Layout.hover row.chromium.png|0| Added|
|Table Coverged - cell actions.In Header Cell.chromium.png|0| Added|
|Table Coverged - cell actions.In Header Cell.hover row.chromium.png|0| Added|

</details>


  <details><summary><b>Table Coverged - headers</b> 18 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table Coverged - headers.Sortable - Dark Mode.chromium.png|0| Added|
|Table Coverged - headers.Sortable - Dark Mode.hover header.chromium.png|0| Added|
|Table Coverged - headers.Sortable - Dark Mode.press header.chromium.png|0| Added|
|Table Coverged - headers.Sortable - High Contrast.chromium.png|0| Added|
|Table Coverged - headers.Sortable - High Contrast.hover header.chromium.png|0| Added|
|Table Coverged - headers.Sortable - High Contrast.press header.chromium.png|0| Added|
|Table Coverged - headers.Sortable Flex Layout - Dark Mode.chromium.png|0| Added|
|Table Coverged - headers.Sortable Flex Layout - Dark Mode.hover header.chromium.png|0| Added|
|Table Coverged - headers.Sortable Flex Layout - Dark Mode.press header.chromium.png|0| Added|
|Table Coverged - headers.Sortable Flex Layout - High Contrast.chromium.png|0| Added|
|Table Coverged - headers.Sortable Flex Layout - High Contrast.hover header.chromium.png|0| Added|
|Table Coverged - headers.Sortable Flex Layout - High Contrast.press header.chromium.png|0| Added|
|Table Coverged - headers.Sortable Flex Layout.chromium.png|0| Added|
|Table Coverged - headers.Sortable Flex Layout.hover header.chromium.png|0| Added|
|Table Coverged - headers.Sortable Flex Layout.press header.chromium.png|0| Added|
|Table Coverged - headers.Sortable.chromium.png|0| Added|
|Table Coverged - headers.Sortable.hover header.chromium.png|0| Added|
|Table Coverged - headers.Sortable.press header.chromium.png|0| Added|

</details>


  <details><summary><b>Table Coverged - resizable columns</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table Coverged - resizable columns.Default.chromium.png|0| Added|
|Table Coverged - resizable columns.End.chromium.png|0| Added|
|Table Coverged - resizable columns.Flex Layout End.chromium.png|0| Added|
|Table Coverged - resizable columns.Flex Layout.chromium.png|0| Added|

</details>


  <details><summary><b>Table Coverged - truncate</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table Coverged - truncate.Default.chromium.png|0| Added|
|Table Coverged - truncate.Flex Layout Truncated.chromium.png|0| Added|
|Table Coverged - truncate.Flex Layout Visible.chromium.png|0| Added|
|Table Coverged - truncate.Flex Layout.chromium.png|0| Added|
|Table Coverged - truncate.Truncated.chromium.png|0| Added|
|Table Coverged - truncate.Visible.chromium.png|0| Added|

</details>


  <details><summary><b>Text Converged</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Text Converged.Typography presets - RTL.normal.chromium.png|48111| Changed|
|Text Converged.Typography presets.normal.chromium.png|58455| Changed|
|Text Converged.Default - RTL.normal.chromium.png|102592| Changed|
|Text Converged.Default - High Contrast.normal.chromium.png|32084| Changed|
|Text Converged.Default - Dark Mode.normal.chromium.png|41505| Changed|
|Text Converged.Default.normal.chromium.png|117578| Changed|

</details>


  <details><summary><b>Toolbar Converged</b> 16 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Toolbar Converged.Default.Toggle On.chromium.png|18345| Changed|
|Toolbar Converged.Default.default.chromium.png|18350| Changed|
|Toolbar Converged.Default.Button Pressed.chromium.png|18491| Changed|
|Toolbar Converged.Large.Toggle On.chromium.png|18345| Changed|
|Toolbar Converged.Large.Button Pressed.chromium.png|18491| Changed|
|Toolbar Converged.Small.Toggle On.chromium.png|15910| Changed|
|Toolbar Converged.Small.Button Pressed.chromium.png|16006| Changed|
|Toolbar Converged.Large.default.chromium.png|18350| Changed|
|Toolbar Converged.Small.default.chromium.png|15919| Changed|
|Toolbar Converged.Vertical Button.Button Pressed.chromium.png|25673| Changed|
|Toolbar Converged.Vertical Button.default.chromium.png|25622| Changed|
|Toolbar Converged.Vertical Button.Toggle On.chromium.png|25475| Changed|
|Toolbar Converged.Vertical.Toggle On.chromium.png|45628| Changed|
|Toolbar Converged.Vertical.default.chromium.png|45634| Changed|
|Toolbar Converged.Vertical.Button Pressed.chromium.png|45832| Changed|
|Toolbar Converged.Far Group.chromium.png|3725| Changed|

</details>


  <details><summary><b>Tooltip Converged</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tooltip Converged.positioning - High Contrast.positioned tooltips.chromium.png|14082| Changed|
|Tooltip Converged.inverted withArrow - Dark Mode.chromium.png|192288| Changed|
|Tooltip Converged.inverted - Dark Mode.chromium.png|192288| Changed|
|Tooltip Converged.positioning - RTL.chromium.png|883| Changed|
|Tooltip Converged.basic - Dark Mode.chromium.png|192288| Changed|
|Tooltip Converged.basic - High Contrast.chromium.png|229128| Changed|
|Tooltip Converged.positioning - High Contrast.chromium.png|583316| Changed|
|Tooltip Converged.withArrow - High Contrast.chromium.png|229127| Changed|
|Tooltip Converged.withArrow - Dark Mode.chromium.png|192288| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

Reply:
similar to previous PR

are those diffs as expected or a bug ?